pkg/interface/src/views/apps/profile/components/SetStatus.tsx

```diff
@@ -50,6 +50,9 @@ export function SetStatus(props: any) {
             value={_status}
             autocomplete='off'
             width='100%'
+            onBlur={() => {
```

Comment:
@liam-fitzgerald This _almost_ does what I want it to, but I need to hook into the `Dropdown` above it to set the status when the user clicks outside the dropdown (I open the dropdown, I set my status, I click outside, it is set). Any suggestions would be extremely welcome

Reply:
In the `useEffect` that you use to set the status, you might consider 
```typescript
return smallControl ? editStatus : () => {}
```